src/Microsoft.DotNet.Web.Spa.ProjectTemplates/content/React-CSharp/Startup.cs

```diff
@@ -1,7 +1,7 @@
 using Microsoft.AspNetCore.Builder;
 using Microsoft.AspNetCore.Hosting;
 using Microsoft.AspNetCore.HttpsPolicy;
-using Microsoft.AspNetCore.SpaServices.ReactDevelopmentServer;
+using Microsoft.AspNetCore.SpaServices.Webpack;
```

Comment:
?

Reply:
Well spotted. That change shouldn't be here, and I don't think the code will currently compile because of it (the extension method on L59 won't be found). Same problem in the ReactRedux template.